manualTests/src/main/x/TestSimple.x

```diff
@@ -2,11 +2,8 @@ module TestSimple {
     @Inject Console console;
 
     void run() {
-        String[] results = new String[];
-        String   next    = "012345";
-
-        results += next[1 ..< 4]; // this used to fail to compile
-        console.print(results);
-
+        String s = "hello    there";
+        console.print(s.split(' ', False));  // old behavior
+        console.print(s.split(' '));         // new behavior
```

Comment:
You could turn this useful test into a TCK test or a permanent unit test. @gridman and I need some test corpus for xunit raw input as well, and every little bit helps.

Reply:
This is not really a test (no invariant is checked), but rather a demonstration. But yes, I'll work on turning every applicable TestSimple submission into a real test when we have a framework

Reply:
Well so are most of the tests in manualTests, and just att the demonstrations as soon as you have them, because we want as much corpus code as possible. Even though JK hasn't figured out exactly what the tests will look like, it should be a trivial operation, so let's every little xtc snippet that is used in debugging to the manual tests repo asap imho.

Reply:
Definitely all the manual tests need to be converted as well. While most of the time it's indeed trivial to convert our "demos" to a proper invariant check, sometimes it's not. Take a look, for example, into services.x. I was thinking about it a bit and could not come up with any decent solution.

Reply:
Yeah, but in this case it's trivial, so slam it in there as TestSplit or something?